### DISCUSSION TITLE
system specification

### DISCUSSION THREAD

**Discussion by Singhamarjeet8130** [facebookresearch/seamless_communication#284] [Q&A] [answered]
what is the best system specification and operating system to run seamless streaming model. If there are any laptop or desktops are there then please list it out.

**Comment by annasun28** [comment, marked as answer]
@Singhamarjeet8130  You can find the recommended hardware in https://huggingface.co/spaces/facebook/seamless-streaming/blob/main/README.md: NVIDIA T4 small (16GB VRAM)
